kolibri/plugins/coach/assets/src/views/reports/ReportsQuizBaseListPage.vue

```diff
@@ -113,6 +157,12 @@
         }
       },
     },
+    $trs: {
+      coachReportsQuizzes: {
```

Comment:
@radinamatic This a string that's used as `aria-label` for this tablist

![Screenshot from 2023-12-13 15-21-12](https://github.com/learningequality/kolibri/assets/13509191/a6cc77d5-5495-4fd6-8680-eb55d3753ae1)

I was thinking if "Coach reports quiz tabs" would make more sense?

Also I think we could improve the string context to say something like "Labels the quiz page tabs in the Coach > Reports > Quizzes"

 I'd be grateful to hear from you before making a final suggestion to @KshitijThareja.

Reply:
Bigger issue here is that since the intention is to merge this into the 0.16 release, we cannot add new strings at this point, as there will be no translations for them available for this release.

Reply:
Ideally the tablist aria-label should say something like ``Report details``, but we have nothing similar on Crowdin. Maybe just reusing the string ``Details`` that we do have in our code already, since this tablist is so deep into Coach and reports features that it would be specific enough.

Reply:
Ah okay, I didn't realize this. Thanks @radinamatic. Yes, then let's go with the existing `'Details'` string for now and I will open a follow-up issue for us to replace it with something better. @KshitijThareja The existing string is saved as `detailsLabel` defined in `commonCoachStrings.js`. Please use it instead of adding a new one. Thank you.

Reply:
And just as a reminder: when writing strings to use in aria-labels for interactive elements, there is no need to include the type of the element as the screen reader already announces it. For example the tablist on the Coach home has the label ``Coach reports``, and the NVDA screen reader announces it as:

`Coach reports  tab control`

If you write the string for aria-label ``Coach reports quiz tabs`` as suggested above,  NVDA would announce it as:

`Coach reports quiz tabs tab control`

To sum it up, no need to put the element type inside its aria-label, it is redundant 🙂

Reply:
❤️🙏🏻

Reply:
I've changed the `aria-label` to use the `detailsLabel` from `commonCoachStrings.js` as asked 👍🏻